bundles/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -472,6 +472,7 @@ private function addAssetNode(ArrayNodeDefinition $rootNode)
                         ->addDefaultsIfNotSet()
                         ->children()
                             ->integerNode('max_pixels')
+                                ->info("Maximum number of pixels an image can have when added (width × height).")
                                 ->defaultValue(40000000)
```

Comment:
Maybe we can improve the formatting of the number a bit. I, at least, have trouble to tell how much it is without the underscores.

```suggestion
                                ->defaultValue(40_000_000)
```